### ISSUE TITLE
My Sats disappeared 

### ISSUE THREAD

**Issue by BitcoinBRC20** [unisat-wallet/extension#100]
I had Trac in my UniSat, and then I listed them on the market with inscription for listing. The listing was verified, and I was able to see my listing in the market.

I woke up in the morning to check if my order was fulfilled, and it was. However, I don't see my Sats, I assume my Bitcoin balance should automatically update, but I don't see the balance for selling my Trac!?

Inscription Number: [#24398668]

Any Help?

**Comment by GGG888GGG** [comment]
@BitcoinBRC20 what's your wallat address?

**Comment by GGG888GGG** [comment]
Is it bc1q69296nydvkz8xrtmj7mcmts0zew3dc7rwgjh8q?
If so, the transaction is d53718142f9d5cc8f1cc14d566cc38ac0831c24ded36a42e8e800a4fe0c2dd3d, you received  0.078btc.

![image](https://github.com/unisat-wallet/extension/assets/132530469/5bd950c4-4bd6-438f-ac85-8353c7d22f8d)

**Comment by BitcoinBRC20** [comment]
That is my public address, correct. However, I don't see that BTC in my wallet and I haven't done any transactions after. Here is a screenshot for reference.

![image](https://github.com/unisat-wallet/extension/assets/142626637/f2024516-8b88-4fd1-927a-bf232fce6dd3)

**Comment by GGG888GGG** [comment]
you may find all your btc transferring records on https://mempool.space/address/bc1q69296nydvkz8xrtmj7mcmts0zew3dc7rwgjh8q

**Comment by GGG888GGG** [comment]
after it, there is a transaction.
![image](https://github.com/unisat-wallet/extension/assets/132530469/63806ee8-281a-4520-b5e8-3521f9e39aae)

**Comment by BitcoinBRC20** [comment]
The address shown (1ESB...ScNT) is the same address that bought TRAC BRC20 contract. Either this transaction shows Trac going to that address in form of "BTC", or the same address somehow hacked my wallet and took my BTC too! 

I didn't send my BTC anywhere, nor I did any other transactions after listing my TRAC. Any thoughts? The problem I couldn't have 2FA or any additional security to the wallet...

**Comment by BitcoinBRC20** [comment]
I have not made any transactions after selling my TRAC on this wallet. I could only think my recovery phrases were leaked when I took a picture of it. However, the picture was never posted or shared and later was deleted. The apps with access to my media are mainly well-known, but could this be how it was leaked? I don't know if there are any methods to resolve this right now or how to ensure my new wallet is not hacked without 2FA or additional security.

I appreciate your assistance

**Comment by GGG888GGG** [comment]
did you access any other website with wallet?

**Comment by BitcoinBRC20** [comment]
Nope, I have not.

**Comment by GGG888GGG** [comment]
according to our experiences:
1. write your private key in a paper or save in a device which never connects to internet, don't save it in any device connected to internet.
2. before visiting any website, be sure it is safe, its site address is completely correct for the official. Fishing websites use similar addresses to confuse users. You'd better visit well-known and formal site. 
3. when sign and fill in the private key, be sure the place is safe.
4. download the wallet from the formal website, like chrome website store. 
5. don't use Origin-Unknown website and wallet software, includes the top ones listed in google search, some phishers also buy paid Listing to confuse users, so you also need to carefully check its site address before click to visit.

**Comment by BitcoinBRC20** [comment]
I followed the above, except for the picture I have mentioned. Is there a way to find out if it's hacked or some kind of an error? And if it is hacked, what can I do at that point?

**Comment by GGG888GGG** [comment]
Sorry, at present we are not sure why you met this event according to the information you provideded, you'd better delete the suspectable software and site, clear cathe, use your anti-virus software to scan the device, and then create/use a new wallet address.

**Comment by BitcoinBRC20** [comment]
This is sad and is very concerning to use Unisat anymore! 

I can't even create my secret phrases, which could also be why my wallet got compromised (I read that some hacks have been happening just because of this). I cannot connect a ledger nano to the exchange, no 2FA.

You have yet to confirm if I was hacked, and I am not even sure if you guys ran any checks on the transactions thread and where that money went on the network. I am genuinely disappointed, and I tried to be patient here. 

I know you tried to help, but I hope the Unisat team will take this matter seriously and further investigate. I shared all the information that there is.

**Comment by truongnghia** [comment]
This is sad and is very concerning to use Unisat anymore!

I can't even create my secret phrases, which could also be why my wallet got compromised (I read that some hacks have been happening just because of this). I cannot connect a ledger nano to the exchange, no 2FA.

You have yet to confirm if I was hacked, and I am not even sure if you guys ran any checks on the transactions thread and where that money went on the network. I am genuinely disappointed, and I tried to be patient here.

I know you tried to help, but I hope the Unisat team will take this matter seriously and further investigate. I shared all the information that there is.

**Comment by GGG888GGG** [comment]
I have already passed the final transaction on to the programmers for analysis. We can only confirm that:
1. The transaction did not use our wallet to connect to phishing websites.
2. The transaction was not sent from our wallet.

No other anomalies were found. If you confirm that you did not initiate the final transaction, it can be confirmed that your wallet has been compromised. We cannot determine the specific reason why you was hacked, but the possibility of mnemonic phrase leakage is more likely.

**Comment by BitcoinBRC20** [comment]
Hi, 

I have done some research, and yes, this transaction was made at the time: 2f4f08f235f9245389e0d616b02bbc610397011a882f593dcabd1bec2b053af6

The buyer (bc1pvh02f4u35mnya80ly8kc8f8k9ewwn7gwnpr0yf6vzkrn2yvl7unswgsjjy) has received Trac tokens, but I never got my Bitcoin in return. 

I don't think my wallet was compromised; I think there is a bug, or the transaction is stuck. I went to Ordiscan and for some reason, the transaction is still being processed ( https://ord.link/24398668), which I think validates what I am sharing here. 

I should be reimbursed for the Trac token or my BTC. I have reached out to the Trac team as well, and they have entirely acknowledged it is an exchange problem.
